modules/social_features/social_profile/config/install/core.entity_view_display.profile.profile.small_teaser.yml

```diff
@@ -11,40 +11,56 @@ dependencies:
     - field.field.profile.profile.field_profile_image
     - field.field.profile.profile.field_profile_interests
     - field.field.profile.profile.field_profile_last_name
+    - field.field.profile.profile.field_profile_nick_name
     - field.field.profile.profile.field_profile_organization
+    - field.field.profile.profile.field_profile_organization_tag
     - field.field.profile.profile.field_profile_phone_number
     - field.field.profile.profile.field_profile_profile_tag
     - field.field.profile.profile.field_profile_self_introduction
     - field.field.profile.profile.field_profile_show_email
-    - image.style.social_small
+    - image.style.social_medium
     - profile.type.profile
   module:
     - image
-_core:
-  default_config_hash: cme1dXyejIMZqleCc1lq6guoASVtyxwnTDJ0TZQPdUA
 id: profile.profile.small_teaser
 targetEntityType: profile
 bundle: profile
 mode: small_teaser
 content:
+  field_profile_function:
+    type: string
+    weight: 1
+    region: content
+    label: hidden
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
   field_profile_image:
     type: image
     weight: 0
     label: hidden
     settings:
-      image_style: social_small
+      image_style: social_medium
       image_link: ''
     third_party_settings: {  }
     region: content
+  field_profile_organization:
+    type: string
+    weight: 2
+    region: content
+    label: hidden
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
 hidden:
   field_profile_address: true
   field_profile_banner_image: true
   field_profile_expertise: true
   field_profile_first_name: true
-  field_profile_function: true
   field_profile_interests: true
   field_profile_last_name: true
-  field_profile_organization: true
+  field_profile_nick_name: true
```

Comment:
@agami4 Please remove lines related with field_profile_nick_name and field_profile_organization_tag as well :)